pkg/dogstatsd/replay/file.go

```diff
@@ -23,7 +23,7 @@ var (
 const (
 	// Version 3+ adds support for nanosecond cadence.
 	// Version 2+ adds support for storing state.
-	datadogFileVersion uint8 = 3
+	datadogFileVersion uint8 = 3 //nolint:staticcheck
 	versionIndex             = 4
```

Comment:
Not sure if there is anything to fix here, if it has to be uint8 then so be it. Will this appease the linter?
```suggestion
)
const (
	versionIndex             = 4
```

Reply:
Added a space in between to avoid the linter. The concrete linter is https://staticcheck.io/docs/checks#SA9004, this is valid solution, but it could also be that the intention was to have everything be `uint8`, which needed more changes.